pima.py

```diff
@@ -290,17 +290,21 @@ def _read_message(self) -> bytes:
     data = bytes([length]) + self._channel.read(length + 2)
     if data == bytes([length]) * len(data):
       raise GarbageInputError('Garbage ({})!'.format(length))
+      sys.exit(1)
```

Comment:
I believe that this Pull Request is not useful because any Python line of code that directly follows a `raise` command will never be executed.  So all those `sys.exit(1)` will never run or make any difference.

Reply:
So I guess that I can just replace `raise` with `print` and then exit?
(So you'll still get the error message, but fully exit instead of just freezing)